cfme/tests/physical_infrastructure/test_redfish_physical_chassis.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.physical.provider.redfish import RedfishProvider
+
+pytestmark = [
+    pytest.mark.provider([RedfishProvider], scope="function"),
+]
+
+
+@pytest.fixture(scope="function")
+def physical_chassis(appliance, provider, setup_provider_funcscope):
+    """Get and return the physical chassis collection."""
+    yield appliance.collections.redfish_physical_chassis.all(provider)
+
+
+def test_redfish_physical_chassis_details_stats(physical_chassis):
```

Comment:
So the fixture is in singluar `physical_chassis` but it returns a list of them... Maybe rename it to `physical_chasis_list` to emphasise it's a list.

Reply:
Actually, you can't tell if "chassis" is singular or plural from the spelling alone. That is why the docstring says it's a collection.